posthog/tasks/verify_persons_data_in_sync.py

```diff
@@ -123,6 +124,11 @@ def _team_integrity_statistics(person_data: List[Any]) -> Counter:
                 properties=pg_person.properties,
                 ch_properties=ch_properties,
             )
+
+        # :KLUDGE: Verify business logic. If versions are in sync so should properties be.
+        if ch_version == pg_person.version and pg_person.properties != ch_properties:
```

Comment:
for it to be more useful currently could we exclude `version == 0`, we should drop that after we're finished with the version async migration on cloud (or we can have both)

```suggestion
        if ch_version != 0 and ch_version == pg_person.version and pg_person.properties != ch_properties:
```

Reply:
I don't think we would remember to but I also don't care for this metric. :sweat_smile: 

This also requires a comment. Mind taking this PR over or creating your own?

Reply:
I'll just mention it in the issue we have for version usage, so I will remember to do this